tests/test_draw_colorclade_tree.py

```diff
@@ -1,10 +1,42 @@
+import functools
+import random
+import typing
+
 from matplotlib import pyplot as plt
 import pandas as pd
+import pytest
+from teeplot import teeplot as tp
```

Comment:
it may already be listed, but do we need to add redeploy as a dependency somewhere?